### PR TITLE
trivial: ci: fix arch PKGBUILD with gitlab.archlinux.org

### PR DESCRIPTION
**Pull request by superm1**
Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [x] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

File: contrib/PKGBUILD [hunk 1]
```diff
@@ -1,45 +1,161 @@
...
-pkgrel=1
-pkgdesc='A system daemon to allow session software to update firmware'
-arch=('i686' 'x86_64')
+pkgbase=fwupd
+pkgname=(
+  fwupd
+  fwupd-docs
+)
+pkgver=1.9.18
+pkgrel=2
```
Comment (superm1) [inline contrib/PKGBUILD:12]: I guess we'll probably want to make these "dummy" again or auto populated in some way?

File: contrib/PKGBUILD [hunk 2]
```diff
@@ -1,45 +1,161 @@
...
+  python-gobject
+  python-pillow
+  vala
+  valgrind
+)
+checkdepends=(umockdev)
+source=(
+  "https://github.com/fwupd/fwupd/releases/download/${pkgver}/${pkgname}-${pkgver}.tar.xz"{,.asc}
+  fwupd.sysusers
+  more-allow-syscall-fixes.patch::https://github.com/fwupd/fwupd/pull/7171.patch
```
Comment (superm1) [inline contrib/PKGBUILD:70]: we'll want to drop this

**Comment by hughsie** [comment]
@superm1 still need this?

**Comment by superm1** [comment]
I'd still like to get to it and fix it, but it's not high priority compared to other stuff.

**Pull request closed without merging**